docs/example/README.md

```diff
@@ -1,24 +1,38 @@
-# OSM Example Usage Guide
+# OSM Demo Guide
 
 ## Table of Contents
+- [Overview](#overview)
 - [Configure Prerequisites](#configure-prerequisites)
 - [Install OSM CLI](#install-osm-cli)
 - [Install OSM Control Plane](#install-osm-control-plane)
-- [Install Applications](#install-applications)
-  - [What Got Installed](#what-got-installed)
-  - [View Application UIs](#view-application-uis)
-- [Access Control Policies](#access-control-policies)
-  - [Going Further](#going-further)
+- [Deploying the Bookstore Demo Applications](#deploying-the-bookstore-demo-applications)
+  - [Create the Bookstore Applications Namespaces](#create-the-bookstore-application-namespaces)
+  - [Onboard the Namespaces to the OSM Mesh](#onboard-the-namespaces-to-the-osm-mesh)
+  - [Deploy the Bookstore Application](#deploy-the-bookstore-application)
+  - [Checkpoint: What got installed?](#checkpoint-what-got-installed)
+  - [View the Applications UIs](#view-the-application-uis)
+- [Deploy SMI Access Control Policies for OSM](#deploy-smi-access-control-policies-for-osm)
+  - [Allowing the Bookthief Application to access the Mesh](#allowing-the-bookthief-application-to-access-the-mesh)
 - [Traffic Encryption](#traffic-encryption)
-- [Traffic Split](#traffic-split-configuration)
+- [Configure Traffic Split between Two Services](#configure-traffic-split-between-two-services)
   - [Deploy v2 of Bookstore](#deploy-v2-of-bookstore)
   - [Update Traffic Split](#update-traffic-split)
-- [Inspect Dashboards](#inspect-dashboards)
+- [Inspect Dashbaords](#inspect-dashboards)
+
+## Overview
+The OSM Demo Guide is designed to quickly allow you to demo and experience the OSM mesh. 
 
 ## Configure Prerequisites
-- Have running Kubernetes cluster
-- Have `kubectl` CLI installed
-- GNU Parallel
+- Kubernetes cluster running Kubernetes v1.15.0 or greater
+- Have `kubectl` CLI installed - [Install and Set Up Kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
+- kubectl current context is configured for the target cluster install
+  - ```kubectl config current-context```
+- Have GNU Parallel installed
+  - ```wget http://ftp.gnu.org/gnu/parallel/parallel-latest.tar.bz2```
+  - ```tar xjf parallel-latest.tar.bz2```
+  - ```cd parallel-20200722```
+  - ```./configure && make```
+  - ```sudo make install```
```

Comment:
These instructions are OS platform specific and should be removed or added per platform.

Reply:
The use of GNU Parallel is only on Linux. Windows uses will use WSL/WSL2.